### DISCUSSION TITLE
User/Device ACL instead of PSK

### DISCUSSION THREAD

**Discussion by nick81nrw** [rustdesk/rustdesk#9883] [Q&A] [answered]
Hi team,

We are currently evaluating Rustdesk Pro as a replacement for Teamviewer. The range of functions and the deployment are working well so far. However, the option to assign user-related devices and connect passwordless is missing. Rustdesk works with fixed pre-shared keys, which in my opinion can also be critical in terms of security. 

We want different technicians to be able to control different devices. In addition, these accesses can change depending on the project. With one PSK per device, the password would have to be shared. In my opinion, control via a user including MFA would even increase security in contrast to the PSK. 
Would it be possible to control access on a user and group basis in an unattended manner?

**Comment by Poordeveloper** [comment, marked as answer]
> Would it be possible to control access on a user and group basis in an unattended manner?

ACL is already there, https://rustdesk.com/docs/en/self-host/rustdesk-server-pro/permissions/
The password and 2FA are additional security layer on the ACL.

I guess you will be interested in this  https://www.reddit.com/r/rustdesk/comments/1gm1bt1/easy_access_equivalent/
